### ISSUE TITLE
Failed to compile some actions for unknown reason

### ISSUE THREAD

**Issue by AlbertSabate** [brisa-build/brisa#362]
For unknown reason brisa build fails to compile some actions when running on the pipeline.

![Screenshot 2024-08-08 at 11 24 19 PM](https://github.com/user-attachments/assets/e4a589d3-5683-46a8-8f96-37b4264e50d3)

You can see RPC returning a 500. According to the error seems the action does not exists. Reviewing the output logs when building I can see many actions failed. No more errors are shown anywhere.

![Screenshot 2024-08-08 at 11 24 59 PM](https://github.com/user-attachments/assets/9a309ffb-6181-46a8-aba4-c354aa4dbf92)

**Comment by AlbertSabate** [comment]
Confirmed is a BunJS issue. https://github.com/oven-sh/bun/issues/2257

I can see on the next version logs they are working on improving transitive dependencies. Hopefully at some points gets fixed :)

**Comment by aralroca** [comment]
@AlbertSabate the issue you shared looks already solved... Maybe is another one?

**Comment by AlbertSabate** [comment]
Nah, they said is likely fixed, not fixed. So I bet they guess it is fixed. Their fix was to add an extra dependency to make it work on GHA, I believe still something going on which nobody is looking at, and I can't provide more information about it because I literally don't know why is happening 😮‍💨.

**Comment by aralroca** [comment]
Create a new issue to Bun then with all the details and linked to this old issue, is the only way to fix this

**Comment by AlbertSabate** [comment]
Let me give it a shot. But when they ask me how to reproduce it or how did it happen, I'm f***ed hehe

**Comment by aralroca** [comment]
We have seen that this bug is an incompatibility with [buildah](https://buildah.io/)

**Comment by AlbertSabate** [comment]
@aralroca we can close this as bunjs has fixed the issue and now is working perfectly in brisa as well.
